brave.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/bash
+export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
+#exec zypak-wrapper /app/extra/brave "$@"
+exec /app/extra/brave --no-sandbox "$@"
```

Comment:
Just a thought, but is it possible to disable the flatpak sandbox instead of the Chromium one?

Reply:
Unfortunately not.  

With flatpak packaging you need to either disable the Chromium sandbox or use zypak to provide a fake sandbox to Chromium (zypak trick uses LD_PRELOAD).

Reply:
I prefer to use zypak because it's a bit more secure and you wouldn't get the `--no-sandbox` warning every time.

Reply:
I disabled Chromium sandbox. If I'm not mistaken it shouldn't have any impact on security because flatpak is already sandboxing everything

Reply:
https://flatpak.org/faq/#Is_Flatpak_compatible_with_other_desktop_isolation_frameworks_

Reply:
Any reason for not using zypak? The commits don't explain anything in that regard

Reply:
@bilelmoussaoui zypak breaks PDF, printing, extensions, and some other things

Reply:
Have you reported that to upstream zypak?

Reply:
yes, it's a known issue

Reply:
Nevermind. Fixed. We're using zypak now!